service/src/test/java/uk/gov/hmcts/reform/fpl/model/CaseDataTest.java

```diff
@@ -68,6 +68,46 @@
     @Autowired
     private Time time;
 
+    @Nested
+    class GetDirectionsToComplyWith {
+
+        @Test
+        void shouldReturnStandardDirectionOrderDirectionsWhenServedCaseManagementOrdersIsEmpty() {
+            List<Element<Direction>> sdoDirections = wrapElements(directionForParty(LOCAL_AUTHORITY));
+
+            CaseData caseData = CaseData.builder()
+                .standardDirectionOrder(Order.builder().directions(sdoDirections).build())
+                .servedCaseManagementOrders(emptyList())
+                .build();
+
+            assertThat(caseData.getDirectionsToComplyWith()).isEqualTo(sdoDirections);
+        }
+
+        @Test
+        void shouldReturnCaseManagementOrderDirectionsWhenServedCaseManagementOrdersIsNotEmpty() {
+            List<Element<Direction>> cmoDirections = wrapElements(directionForParty(LOCAL_AUTHORITY));
+            List<Element<Direction>> sdoDirections = wrapElements(directionForParty(CAFCASS));
+
+            CaseData caseData = CaseData.builder()
```

Comment:
case data is build here and in method above.. could be extracted and just pass in list as param